src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -150,18 +165,19 @@ public static NumberPropertyDefinition number(String code) {
    * @param label the label of the property definition
    * @return a new NumberPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Également : 
```suggestion
  Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use number(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```
```